### PR TITLE
Hiding 'Use Ticketing' Toggle to prevent spams

### LINKED ISSUE #2482: Hide option "use ticketing system" to prevent spam and require tickets to make event live
Please hide the option "use ticketing system" and only show the option with tickets. Only allow to save an event if a ticket has been added.

Screenshot from 2019-04-06 20-37-42

Background:

The option "use ticketing system" is a nice feature to add events of external websites. The options has been abused though to add spam events.

Currently we don't have the resources to implement a thought-through approach and configuration to prevent this.

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Hidden the toggle and added a simple description.

Fixes #2482

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 Review

**Comment by Anupam-dagar** [comment]
@kushthedude you missed one part of the issue. The form should get submitted if atleast one ticket is added if ticketing system is enabled. I think this fix will allow 0 tickets to be saved also.

**Comment by kushthedude** [comment]
@Anupam-dagar Thanks for pointing out will look into it.

**Comment by kushthedude** [comment]
@CosmicCoder96 Review

**Comment by Anupam-dagar** [comment]
@kushthedude this just writes an info about adding atleast 1 ticket but doesn't perform any real checks.

File: app/templates/components/forms/wizard/basic-details-step.hbs [hunk 1]
```diff
@@ -550,10 +556,12 @@
       <i class="save icon"></i>
     </button>
     {{#if data.event.locationName}}
-      <button class="green ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'publish'}}>
-        {{t 'Publish'}}
-        <i class="check icon"></i>
-      </button>
+      {{#if data.event.tickets.length}}
```
Comment (kushthedude) [inline app/templates/components/forms/wizard/basic-details-step.hbs]: @Anupam-dagar See here

**Comment by kushthedude** [comment]
@Anupam-dagar refer https://github.com/fossasia/open-event-frontend/pull/2518/files#r273325443

**Comment by Anupam-dagar** [comment]
With this `If`, if I disable the ticket system then I won't ever be able to create the event.

**Comment by kushthedude** [comment]
@Anupam-dagar I think thats the whole point of issue , the event should only be published if there is ticket available to prevent spams

**Comment by Anupam-dagar** [comment]
@kushthedude disabling ticketing system happens from the settings. If that is disabled then user should be allowed to submit but if the ticketing is enabled then there should be atleast one ticket.

**Comment by kushthedude** [comment]
@Anupam-dagar i understood your point but what i am saying is if you disabled the module for ticketing than also spam can happen using external link , thats why i had put this condition so that there should a ticket to publish the event or else event should not be published

**Comment by Anupam-dagar** [comment]
I guess the issue is there to remove that external link dependency only so that if somebody has to use ticketing system then they have to use open event's ticketing system and not their external ticket system.

**Comment by kushthedude** [comment]
Any updates on it @CosmicCoder96

**Comment by kushthedude** [comment]
@mariobehling Are this changes needed or the PR should be closed #2482

**Comment by kushthedude** [comment]
@mariobehling Do we need this changes ?

**Comment by CosmicCoder96** [comment]
https://github.com/fossasia/open-event-frontend/issues/2482#issuecomment-491208474

**Comment by kushthedude** [comment]
@CosmicCoder96 since #2482 is reopened , I request you to reopen this PR .

**Pull request closed without merging**